app-text/cherrytree/cherrytree-0.99.52.ebuild

```diff
@@ -30,8 +32,7 @@ IUSE="nls test"
 # Has deps that aren't available in ::gentoo repo
 RESTRICT="test"
 
-RDEPEND="
-	app-i18n/uchardet
+RDEPEND="app-i18n/uchardet
```

Comment:
Why did you change it?

Reply:
To match the rest of the dependency declarations?